src/components/App.tsx

```diff
@@ -39,6 +39,25 @@ class App extends React.Component<Record<string, unknown>, AppState> {
     // TODO: fetch real ETH price
     const ethPriceUsd = 2700;
     this.fetchAave(this.mainAccount, ethPriceUsd);
+
+    const httpProvider = this.web3.eth.providers["HttpProvider"];
```

Comment:
confirm

Reply:
getting an error if I do this
![image](https://user-images.githubusercontent.com/6494625/116797005-ed1ecd80-aa95-11eb-8eff-c7f33414cfc3.png)

Reply:
can try some different providers
example: https://github.com/bentiancai629/solidity-demo/blob/2d7aba858d5ad1415d1027735d867609dd21673d/aave/usage.js